src/paras/Assignment_30/FindSecondMaxFromArr.java

```diff
@@ -0,0 +1,29 @@
+/*
+Assignment - 30 : 4th May'2021
+
+WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+output : 55
+ */
+package paras.Assignment_30;
+
+public class FindSecondMaxFromArr {
+	
+	void findSecondMax(int [] arr) {
+		int maxNumber = arr[0];
+		int secondMax = arr[0];
+		for(int index = 1; index < arr.length; index++) {
```

Comment:
this will fail for below input
int[] arr = { 112, 93, 10, 33, 43, 111, 55, 97, 11, 3, 88, 99, 113 };

logic is write it should be like below
<img width="394" alt="Screenshot 2021-05-04 at 9 12 19 PM" src="https://user-images.githubusercontent.com/58640315/117030873-ae02ad80-ad1d-11eb-9193-2a7b471efe37.png">